source/1_engine/rex_engine/src/cmd_line_args.cpp

```diff
@@ -18,6 +18,11 @@ namespace rex
     parse_cmd_line(rsl::string_view(cmdLine));
   }
 
+  const rsl::vector<rsl::string_view>& CommandLineArguments::arguments() const
+  {
+    return m_arguments;
```

Comment:
just to note, this will be removed in the command_line_args branch

Reply:
but command line arguments will be parsed better (imo). maintained better and you can log them to the console, so they'll document themselves